chia/consensus/default_constants.py

```diff
@@ -38,8 +38,8 @@
         "3d8765d3a597ec1d99663f6c9816d915b9f68613ac94009884c4addaefcce6af"
     ),
     "MAX_VDF_WITNESS_SIZE": 64,
-    # Size of mempool = 50x the size of block
-    "MEMPOOL_BLOCK_BUFFER": 50,
+    # Size of mempool = 50x the size of block # temporary change until #9125 gets in
+    "MEMPOOL_BLOCK_BUFFER": 10,
```

Comment:
PR9125 is already merged into main. Is it still valid that this temporary fix should be removed than?

Reply:
@Yostra not sure if commenting on already closed PRs is tracked, so I'm @

Reply:
Yes this temp fix should be removed